portfolio/.theia/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
```

Comment:
Optional: These lines can be removed if you feel like they won't be useful later

Reply:
Sure. Thanks.